etabotsite/etabotsite/settings.py

```diff
@@ -153,4 +152,4 @@
 #SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 #SECURE_SSL_REDIRECT = True
 #SESSION_COOKIE_SECURE = True
-#CSRF_COOKIE_SECURE = True
+CSRF_COOKIE_SECURE = False
```

Comment:
This is not secure to make it false in production.

Reply:
this is to allow csrf cookies to be sent over http - this is temporary while we are setting up HTTPS